react-native/react/constants/update.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+
+import type {TypedAction} from '../constants/types/flux'
+import type {Asset, UpdateType} from '../constants/types/flow-types'
+
+export const showUpdatePrompt = 'update:showUpdatePrompt'
+export type ShowUpdateAction = TypedAction<'update:showUpdatePrompt', showUpdatePromptPayload, UpdateError>
+
+export const registerUpdateListener = 'update:registerUpdateListener'
+export type RegisterUpdateListenerAction = TypedAction<'update:registerUpdateListener', any, any>
+
+export const onCancel = 'update:onCancel'
+export type OnCancelAction = TypedAction<'update:onCancel', any, any>
+export const onSkip = 'update:onSkip'
+export type OnSkipAction = TypedAction<'update:onSkip', any, any>
+export const onSnooze = 'update:onSnooze'
+export type OnSnoozeAction = TypedAction<'update:onSnooze', any, any>
+export const onUpdate = 'update:onUpdate'
+export type OnUpdateAction = TypedAction<'update:onUpdate', any, any>
+
+export type UpdateActions = ShowUpdateAction | RegisterUpdateListenerAction
```

Comment:
Shouldn't this be `type UpdateActions = ShowUpdateAction | RegisterUpdateListenerAction | OnSkipAction | OnSnoozeAction | OnUpdateAction | OnCancelAction` to include all the action types. Or did flow give you some weird error when you tried that?